eras/babbage/impl/cardano-ledger-babbage.cabal

```diff
@@ -144,6 +144,7 @@ library testlib
     generic-random,
     heredoc,
     microlens,
+    microlens-ghc,
```

Comment:
This has a small dependency footprint and doesn't bring in any new dependencies

Reply:
It's needed only to get instances for `containers`

Reply:
Not a big fan of using lenses on data structures, but `at` used on `Map` isn't the worse one from the available operations :smile:

Reply:
I figured I might be able to get away with that :grin: